qiskit/aqua/algorithms/amplitude_amplifiers/grover.py

```diff
@@ -386,7 +386,7 @@ def _run(self) -> 'GroverResult':
         # in ``rotation_counts``. Once a good state is found (oracle_evaluation is True), stop.
         for power in self._iterations:
             if self._sample_from_iterations:
-                power = self.random.integers(power) + 1
+                power = self.random.integers(power)
```

Comment:
Can you update the docstring saying that if `sample_from_iterations` is True then a sample will be drawn that is *smaller* than the iteration?